content/webapp/services/prismic/transformers/body.ts

```diff
@@ -511,6 +490,9 @@ export function transformBody(body: Body): BodySlice[] {
 
         case 'mediaObjectList':
           return transformMediaObjectListSlice(slice);
+
+        default:
+          return undefined;
```

Comment:
linting fix: `Array.prototype.map() expects a value to be returned at the end of arrow function`